src/Compilers/Shared/ShadowCopyAnalyzerAssemblyLoader.cs

```diff
@@ -75,13 +75,7 @@ private void DeleteLeftoverDirectories()
             }
         }
 
-        [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Reliability", "CA2001:AvoidCallingProblematicMethods",
```

Comment:
We run FXCop as part of division exit criteria near each release. Tagging @shyamnamboodiripad who used to do it earlier, I am not sure who does this now.